src/cr/cube/matrix/measure.py

```diff
@@ -50,6 +52,16 @@ def means(self):
         """_Means measure object for this cube-result"""
         return _Means(self._dimensions, self, self._cube_measures)
 
+    def get_t_stats_for(self, subvar_idx):
```

Comment:
Why this `get_t_stats_for` ? It smells like a Java property 😂 . Why not `t_stats` ? Or even, as mentioned before, wrap tstats and pvals withiun a single functions that have pvals and zscore, and then just a `pairwise_t_test`  funciont where we pass the subvar_idx and returns both t-tests values? If i well remember these values are mainly used in the exporter for the color shading of the cells for rendering the significance. So we always need them both for doing stuff ... i'm inclined to think that having 2 separate function is not worthwhile.

Reply:
Yeah, this is the complicated part... I need to have a measure "per subvariable". How to do that is a question... I need a matrix for each subvariable, essentially making the result of the measure 3D. But I can only apply the concept of `blocks` on a 2D. So I need to iterate. And the question is: where I do the iteration so it makes the most sense? In this approach, I've tried to iterate over subvars, and create a measure for each one of them, so that the measure itself can behave like all other measures do. And then assembler just wraps those matrices together.

I'll try to do something better... I'll see...

Reply:
fixed the naming, but left them as functions. resolving conversation so that I can attend to other comments.